app/resources/views/pages/location_management/suggestion_list.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.default')
+@section('content')
+<div class="suggestion-list">
+	<div class="text-center">
+		<h1><a href="/location/management/my-locations">Suggesstions for {{ $name }}</a></h1>
+	</div>
```

Comment:
When no suggestions are on a specific location, it would be nice to have a message saying that.

![image](https://user-images.githubusercontent.com/11838191/43256902-3b22bd64-909c-11e8-85ba-d91f8cf6a92a.png)

That would be consistent with how My locations:
![image](https://user-images.githubusercontent.com/11838191/43256948-5d040118-909c-11e8-89a1-78c5698c6d41.png)